src/__tests__/animation.js

```diff
@@ -0,0 +1,36 @@
+import transformCss from '..'
+
+const emptyAnimation = {
+  animationName: [],
+  animationDuration: [],
+  animationTimingFunction: [],
+  animationDelay: [],
+  animationIterationCount: [],
+  animationDirection: [],
+  animationFillMode: [],
+  animationPlayState: [],
+}
+
+it('animation none', () => {
+  expect(transformCss([['animation', 'none']])).toEqual(emptyAnimation)
+})
+
+it('animation a single property', () => {
+  expect(transformCss([['animation', 'slide-in linear .3s']])).toEqual({
+    animationName: ['slide-in'],
```

Comment:
I think it'd be pretty great to support this stuff in this library

I have a slight issue with mixing RN styles with things RN cannot handle

Perhaps we could change the API to return `{ styles, animation, transition }` etc. where `styles` is all the stuff that's compatible with RN, then all the other properties are things libraries can use to polyfill support etc.